examples/platform/linux/AppMain.cpp

```diff
@@ -109,6 +109,7 @@ void EventHandler(const DeviceLayer::ChipDeviceEvent * event, intptr_t arg)
     }
 }
 
+#if !defined(ENABLE_CHIP_SHELL)
```

Comment:
Should probably document why this is code is disabled if ENABLE_CHIP_SHELL.

Reply:
added comment